tests/test_motion_service.py

```diff
@@ -17,7 +17,8 @@
     "gantry": PoseInFrame(reference_frame="gantry", pose=Pose(x=2, y=3, z=4, o_x=3, o_y=4, o_z=5, theta=21)),
 }
 MOTION_CONFIGURATION = MotionConfiguration(
-    vision_services=[ResourceName(namespace="rdk", type="service", subtype="vision", name="viz1")],
+    obstacle_detectors=[ObstacleDetector(vision_service=ResourceName(namespace="rdk", type="service", subtype="vision", name="viz1"),
+                                         camera=ResourceName(namespace="rdk", type="component", subtype="camera", name="cam1"))],
```

Comment:
I assume it's fine to just use a random camera here in the test?

Reply:
Anything that can return `GetObjectPointCluds` for an obstacle. which cameras do.